### PR TITLE
Updated roadmap

### PR DESCRIPTION
**Pull request by norro**
* deleted deprecated points (2018, 2019)
* moved remaining points to 2020

@ralph-lange anything else we should add/kick? maybe something relating an intralogistics use-case?

### CODE REVIEW

File: _docs/concepts/client_library/system_modes/index.md [hunk 1]
```diff
@@ -108,18 +108,11 @@ A simple example is provided at:
 
 ## Roadmap
 
-**2018**
-*   Extended lifecycle concept and implementation for ROS 2 and micro-ROS.
-*   Modeling concept to specify system hierarchy as well as system modes of systems, subsystems, and their mapping along the system hierarchy down to nodes.
-*   Mode inference and mode manager in C++ for ROS 2.
```
Comment (ralph-lange) [inline _docs/concepts/client_library/system_modes/index.md]: In 2018 only a Python prototype. The definite implementation was mainly in 2019. Please also add diagnostics port from ROS 1 to ROS 2 to 2019. Finally, you could mention the design of the rules concept/mechanisms in 2019.

File: _docs/concepts/client_library/system_modes/index.md [hunk 2]
```diff
@@ -108,18 +108,11 @@ A simple example is provided at:
...
-
-**2019**
+**2020**
 *   Specific implementation of mode manager for micro-ROS as may be necessary.
```
Comment (ralph-lange) [inline _docs/concepts/client_library/system_modes/index.md]: Please remove this line. It is sufficient to have a mode manager on the main uP for now.

File: _docs/concepts/client_library/system_modes/index.md [hunk 3]
```diff
@@ -108,18 +108,11 @@ A simple example is provided at:
...
 *   Diagnostics framework for micro-ROS, interoperating with ROS 2 diagnostics.
```
Comment (ralph-lange) [inline _docs/concepts/client_library/system_modes/index.md]: Please write: Port of basic diagnostics functionalities from ROS 2 to micro-ROS.

File: _docs/concepts/client_library/system_modes/index.md [hunk 4]
```diff
@@ -108,18 +108,11 @@ A simple example is provided at:
 
 ## Roadmap
 
-**2018**
-*   Extended lifecycle concept and implementation for ROS 2 and micro-ROS.
-*   Modeling concept to specify system hierarchy as well as system modes of systems, subsystems, and their mapping along the system hierarchy down to nodes.
-*   Mode inference and mode manager in C++ for ROS 2.
-
-**2019**
```
Comment (ralph-lange) [inline _docs/concepts/client_library/system_modes/index.md]: 2019 must not be removed completely.

**Comment by norro** [comment]
I thought we might want to skip 2018/2019 as it is no longer on the road ahead. :D But I see your point, I sorted and cleaned 2018 - 2019 as well.

The diff somehow looks really mixed up, it might be easier to look at the branch itself: [fix/system-modes-roadmap:system_modes/index.md#roadmap](https://github.com/micro-ROS/micro-ROS.github.io/blob/fix/system_modes_roadmap/_docs/concepts/client_library/system_modes/index.md#roadmap)

**Comment by norro** [comment]
@ralph-lange also, I added lightweight "recovery mechanisms" in 2020, hintign at the bottom-up rules thingy.

**Pull request merged**